libraries/provider_firewall_rule_iptables.rb

```diff
@@ -92,7 +122,15 @@ def build_firewall_rule(type = nil)
         if [:pre, :post].include?(new_resource.direction)
           firewall_rule << '-t nat '
         end
+
+        # Iptables order of prameters is important here see example output below:
```

Comment:
Hi Martin, in my tests on rhel, the output is static, no matter what the order of your input

example:
iptables -A INPUT -p tcp -m tcp --sport 80 -m state --state NEW -i eth -d 127.0.0.2 -s 127.0.0.8 -j ACCEPT
results in a iptables-save output of:
-A INPUT -s 127.0.0.8/32 -d 127.0.0.2/32 -i eth -p tcp -m tcp --sport 80 -m state --state NEW -j ACCEPT
not sure if this is RHEL specific, but on rhel 6.5 this was the case.